modules/python/common.cmake

```diff
@@ -85,7 +110,7 @@ set_target_properties(${the_module} PROPERTIES
                       LIBRARY_OUTPUT_DIRECTORY  "${OPENCV_PYTHON_EXTENSION_BUILD_PATH}"
                       ARCHIVE_OUTPUT_NAME ${the_module}  # prevent name conflict for python2/3 outputs
                       PREFIX ""
-                      OUTPUT_NAME cv2
+                      OUTPUT_NAME _cv2
```

Comment:
> install/lib/python2.7/dist-packages/cv2/python-2.7/_cv2.so
> install/lib/python3.5/dist-packages/cv2/python-3.5/_cv2.cpython-35m-x86_64-linux-gnu.so

> _cv2

This would break already existed packaging scripts.

Reply:
I've got your point. I've reverted this.